src/documentation/pages/components/footer/examples/_external-lumen-portal.vue

```diff
@@ -0,0 +1,530 @@
+<template lang="pug">
+<ComponentExample title="External" id="external-lumen-portal" :tabs="exampleTabs">
```

Comment:
Check this identation in every example.

```suggestion
<template lang="pug">
  <ComponentExample title="External" id="external-lumen-portal" :tabs="exampleTabs">
```